modules/meeting/app/forms/meeting_agenda_item/duration.rb

```diff
@@ -34,6 +34,7 @@ class MeetingAgendaItem::Duration < ApplicationForm
       label: MeetingAgendaItem.human_attribute_name(:duration_in_minutes),
       leading_visual: { icon: :stopwatch },
       visually_hide_label: true,
+      max: 86400,
```

Comment:
That's 60 days. Is it intended? Did you mean `24*60` instead? (`24.hours.in_minutes.to_i`)